front/src/applications/operationalStudies/hooks/usePathProjection.ts

```diff
@@ -4,33 +4,60 @@ import { useSelector } from 'react-redux';
 
 import { osrdEditoastApi, type InfraWithState } from 'common/api/osrdEditoastApi';
 import usePathProperties from 'modules/pathfinding/hooks/usePathProperties';
-import type { TrainScheduleId } from 'reducers/osrdconf/types';
 import { getTrainIdUsedForProjection } from 'reducers/simulationResults/selectors';
-import { formatTrainScheduleIdToEditoastTrainId } from 'utils/trainId';
+import {
+  formatPacedTrainIdToEditoastTrainId,
+  formatTrainScheduleIdToEditoastTrainId,
+  isPacedTrain,
+  isTrainSchedule,
+} from 'utils/trainId';
 
 const usePathProjection = (infra: InfraWithState) => {
   const trainIdUsedForProjection = useSelector(getTrainIdUsedForProjection);
 
```

Comment:
Small suggestion to improve readability with  a more explicit variable:
```suggestion
const isTrainScheduleProjection = trainIdUsedForProjection && isTrainSchedule(trainIdUsedForProjection);
```
The variable could be reused in several places:
- in skip condition in editoast calls
- condition in path memo

What do you think ?

Reply:
Yes i'll change it